e2e/src/tests/rpc_queries.rs

```diff
@@ -191,7 +199,10 @@ pub async fn rpc_query_proof_test_2() -> Result<()> {
 	let cells = Cells::try_from(vec![cell.clone()]).unwrap();
 
 	// RPC call
-	let actual_proof: Vec<GDataProof> = client.rpc_methods().query_proof(cells, block_hash).await?;
+	let mut params = RpcParams::new();
+	params.push(cells)?;
+	params.push(block_hash)?;
+	let actual_proof: Vec<GDataProof> = client.rpc().request("kate_queryProof", params).await?;
```

Comment:
As side-effect, this also remove the capability of use the RPC with type safety.
I think it is not a good a Rusty-approach, because any future change in RPC on future release will be hardly detected on Rust clients.